uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/task-list.js

```diff
@@ -210,16 +210,11 @@ define([
             };
 
             $scope.updateTask = function(task, updateObj) {
-
-                var updatePromise = TaskService.save(angular.extend({
-                    id: task.id
-                },updateObj));
-
-                updatePromise.then(null,function(reason){
-                    CRM.alert(reason, 'Error', 'error');
+              return TaskService
+                .save(angular.extend({}, task, updateObj))
+                .catch(function (reason) {
+                  CRM.alert(reason, 'Error', 'error');
```

Comment:
Instead the having same "Error" twice, can we mention method name in the 2nd/last parameter?